roles/splunk_common/tasks/install_splunk_tgz.yml

```diff
@@ -1,14 +1,29 @@
-- name: Install Splunk (Linux)
+- name: Install Splunk (Linux) from network
   unarchive:
     src: "{{ splunk.build_location }}"
     dest: "{{ splunk.opt }}"
     owner: "{{ splunk.user }}"
     group: "{{ splunk.group }}"
     remote_src: "{{ splunk.build_remote_src }}"
+  when: splunk.build_remote_src
   register: install_result
   until: install_result is succeeded
   retries: "{{ retry_num }}"
   delay: 3
   become: yes
   become_user: "{{ privileged_user }}"
 
+- name: Install Splunk (Linux) from local build
+  unarchive:
+    src: "{{ '/opt/container_artifact/' + splunk.build_location|basename }}"
```

Comment:
I have some issues with this fix as it stands now.  With this string hardcoded, its assumed that this directory will always exist, when that may not be the case depending on the platform and the environment.

Reply:
fixed hard code issue, and using regex filter to remove 'file://'